tests/unit/testtasksentry.cpp

```diff
@@ -0,0 +1,169 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "testtasksentry.h"
+#include "../../src/settingsholder.h"
+
+#include <QTextStream>
+
+#include "../../src/tasks/sentry/tasksentry.h"
+#include "helper.h"
+
+void TestTaskSentry::envelopeWithJustAnAttachment() {
```

Comment:
use the `_data()` slot for this. See other tests.

Reply:
done :)